PowerEditor/src/NppLibsVersion.h

```diff
@@ -0,0 +1,5 @@
+// NppLibsVersion.h
+// - generated by NppLibsVersionH-generator.bat
+#define NPP_SCINTILLA_VERSION "5.5.4"
+#define NPP_LEXILLA_VERSION "5.4.2"
+#define NPP_BOOST_REGEX_VERSION "1_85"
```

Comment:
That means each update I have to modify these 3 values.
I'm not sure that I'm ready for such effort.

Reply:
@donho 

`// - generated by NppLibsVersionH-generator.bat`

`generated`, not manually updated/modified...

The only thing that currently holds this PR in draft-stage is precisely the fact that it is not yet fully automated. MSVS builds are already covered ok but I still did not find out the right way/path for the GCC makefile builds. My own local installation of msys2/mingw works ok but the GitHub one is somewhat different.

Reply:
@xomx 
If the file "NppLibsVersion.h" is generated automatically during the build, then this file should not be in the repository. I suppose that it's here temporarily for the aim of the review, right?

Reply:
@donho 
> this file should not be in the repository. I suppose that it's here temporarily for the aim of the review, right?

No, "NppLibsVersion.h" has to be already part of the project files due to the N++ `CMakeLists.txt` (otherwise it will be complicated ^2 because of you will need updating that file too at the pre-build time...)

But you have a point - the comment inside the header file should be rather tagged as `// - maintained by NppLibsVersionH-generator.bat` than `generated`.